samples/peripheral/radio_test/src/radio_test.h

```diff
@@ -54,12 +54,12 @@ enum radio_test_mode {
 	MODULATED_TX_DUTY_CYCLE,
 };
 
-/**@brief Radio test nRF21540 configuration */
-struct radio_test_nrf21540 {
-	/* nRF21540 activation delay. */
+/**@brief Radio test Front-end Module (FEM) configuration */
```

Comment:
Also in this header file, please decapitalize -> front-end module in all places

Reply:
Done. Thanks for checking that